src/App.css

```diff
@@ -1,26 +1,64 @@
 body {
   margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: Montserrat, sans-serif;
```

Comment:
- [ ] Please According to the project requirement, Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)
- Please make these in order to stick to the original design:

![petit0](https://user-images.githubusercontent.com/79058364/162425998-4fd6510f-30ed-4276-90ee-13cedf4851ee.png)
![petit1](https://user-images.githubusercontent.com/79058364/162426005-094f46b5-4328-4744-b879-dd84af147f04.png)
![petit2](https://user-images.githubusercontent.com/79058364/162426009-50b58674-7a1b-4744-b6b7-ced42dc88700.png)

- Figma design

![bookstore](https://user-images.githubusercontent.com/79058364/162426330-10521c8c-526f-4fca-9450-dc3154d41b55.png)